hknweb/settings/common.py

```diff
@@ -78,6 +78,8 @@
                 'django.contrib.messages.context_processors.messages',
                 'social_django.context_processors.backends',
                 'social_django.context_processors.login_redirect',
+                'social_django.context_processors.login_redirect',
+                'hknweb.views.users.add_officer_context',
```

Comment:
So let me get this straight...for some reason, these functions are called as "context processors" which allow variables like `viewer_is_an_officer` to be referenced in templates like `base.html`? Sounds pretty confusing, but I'm not sure of a better solution. Maybe ask @jameslzhu on this one since I've never dealt with context processors before.

Reply:
The other possible way is to use template tags, but that seems to require creating a new folder and two new files, just for one function